pygments/lexers/_openedge_builtins.py

```diff
@@ -591,9 +606,13 @@
     'DROP-DOWN-LIST',
     'DROP-FILE-NOTIFY',
     'DROP-TARGET',
+    'ds-close-cursor',
```

Comment:
why is this lowercased?

Reply:
Its like that in the documentation, i fixed it anyway to stay consistent